src/lib/airtable/request.js

```diff
@@ -271,6 +274,32 @@ export const createManyPurchaseRequests = async (records) => {
   return Promise.all(createPromises);
 };
 
+// NONGENERATED: Create an Inventory Update and update the inventory's current qty
+// TODO: handle offline workflow of creating inventory updates for inventory
+// that was created offline (no Airtable id).
+export const createInventoryUpdateAndUpdateInventory = async (userId, inventory, updatedAmount ) => {
```

Comment:
Since this is used in multiple places, I ended up just moving all the `inventoryUpdate` creation work into here. Is that bad practice?

Also this function name is a nightmare and I'm definitely open to suggestions 😅

Reply:
Kinda bad practice because we don't use this convention anywhere else, but as long as you put comments in here about why you're doing it this way I'm down to do this since I see the upside also if it's being used in multiple places.